### PR TITLE
Merge 1.7.8.2-build in 1.7.8.x [merge]

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Merge 1.7.8.2-build in 1.7.8.x
| Type?             | improvement
| Category?         | ME
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| How to test?      | CI is green
| Possible impacts? | 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26908)

### CODE REVIEW

File: app/AppKernel.php [hunk 1]
```diff
@@ -38,6 +38,7 @@ class AppKernel extends Kernel
     const MINOR_VERSION = 8;
     const RELEASE_VERSION = 3;
 
+
```
Comment (Progi1984) [inline app/AppKernel.php]:
```suggestion
```

**Review by jolelievre — COMMENTED** [review]
Legit except the extra line in kernel

File: app/AppKernel.php (hunk 1 above)
Comment (matks) [reply app/AppKernel.php]: git conflict 😭  leftover

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @matks

**Pull request merged**